### DISCUSSION TITLE
Configure ConfirmationURL?

### DISCUSSION THREAD

**Discussion by scarsam** [supabase/supabase#595] [Questions] [answered]
👋 Would it be possible to change the `{{ .ConfirmationURL }}` somehow? Right now it's always taking me to the root path `/`.

```
<h2>Confirm your signup</h2>

<p>Follow this link to confirm your user:</p>
<p><a href="{{ .ConfirmationURL }}">Confirm your mail</a></p>
```

I'm trying to build out an onboarding experience and would like to have the email take the user into onboarding directly, instead of root, and then have to redirect from root to my onboarding URL.

Thanks for some awesome software so far!

**Comment by gomflo** [comment, marked as answer]
Yep, go to `Authentication` -> `Settings` -> `SITE_URL`

**Comment by scarsam** [comment]
Thank you, @gomflo changing the `SITE_URL` doesn't seem to have any effect on the confirmation email getting sent out once I signup locally. It still redirects me to `localhost:3000/` when I click the link from the email.

**Reply by scarsam** [reply]
I believe you're right, I just had some server-side logic that caused some redirect issues on my end. `SITE_URL` should be correct.

**Comment by slavicdev34** [comment]
You can use different variables inside your email template. I checked https://github.com/supabase/gotrue/blob/8cb3d71a8c3fcbd9fd0e23586f1203d5861a56a5/mailer/template.go#L94-L98
e.g. you can redirect user to {{ .SiteURL }}/api/account?confirmationToken={{ .Token }}
